src/main/java/uk/gov/ons/ctp/response/sample/ingest/CsvIngesterSocial.java

```diff
@@ -65,6 +65,7 @@ public SampleSummary ingest(MultipartFile file)
     SampleSummary sampleSummary;
     SocialSurveySample businessSurveySample = new SocialSurveySample();
     List<SocialSampleUnit> samplingUnitList = new ArrayList<>();
+    Integer expectedCI = 1;
```

Comment:
Why did you use Integer expectedCI = 1. Please give more details in comments about this hardcoded value